src/app/components/Select/index.tsx

```diff
@@ -96,6 +97,54 @@ const TertiaryButton = forwardRef(
   },
 )
 
+export const StyledLightSelectButton = styled(StyledSelectButton)(() => ({
```

Comment:
Not a fan of this, creating new components, rather I would add some kind of prop to `StyledSelectButton`, that would change the styling based on that via `slotProps`.

Reply:
I did this now, but I am not sure if that made the code simpler or not.